CMakeLists.txt

```diff
@@ -12,16 +12,23 @@ set(GCF_IGNORE_LIST "" CACHE STRING
     "Semi colon separated list of directories to ignore")
 set(GCF_CLANGFORMAT_MINIMAL_VERSION "0.0.0" CACHE STRING
     "The minimal required version of clang-format")
-set(GCF_FORCE_OVERWRITE OFF CACHE BOOL
-    "If true, always overwrite pre-commit hook and script")
+if(NOT GCF_FORCE_OVERWRITE)
+  set(GCF_FORCE_OVERWRITE OFF CACHE BOOL
+      "If true, always overwrite pre-commit hook and script")
+endif()
```

Comment:
maybe `option` is the right thing to use here?

Reply:
didn't know that exist :sweat_smile: thanks!